pkg/encoder/encoder.go

```diff
@@ -184,13 +184,27 @@ func (p *CompactEncoder) EventToString(response *tetragon.GetEventsResponse) (st
 			}
 			return CapTrailorPrinter(fmt.Sprintf("%s %s %s %v", event, processInfo, file, bytes), caps), nil
 		case "fd_install":
-			event := p.Colorer.Blue.Sprintf("📬 %-7s", "open")
+			event := p.Colorer.Blue.Sprintf("📬 %-7s", "fd_install")
```

Comment:
We dd this intentionally with the idea that fd_install is the correct hook to monitor for file "open" events.

Reply:
put this in its own PR if we want to debate this. Then we can merge the rest of this more or less now.